app/assets/stylesheets/active_admin/_forms.scss

```diff
@@ -79,169 +109,103 @@ form {
-        label { display: none; }
-        input { display:inline; margin:0; padding:0;  }
-      }
-    }
+  /* BOOLEAN OVERRIDES */
 
-    /* Check Boxes or Radio fields */
-    &.check_boxes, &.radio {
-      fieldset ol {
-        margin-bottom:-0.6em;
-        li {
-          margin:0.1em 0 0.5em 0;
-          label {
-            float:none;
-            width:100%;
-            input { margin-right:0.2em; }
-          }
-        }
-      }
-    }
+  .boolean input,
+  .choice input {
+    margin: 0 0.3em 0 0;
+  }
 
-    /* Boolean Field */
-    &.boolean {
-      height: 1.1em;
-      label {
-        width: 80%;
-        padding-left:20%;
-        padding-right: 10px;
-        text-transform: none !important;
-        font-weight: normal;
-        input { margin:0 0.5em 0 0.2em; }
-      }
-    }
+  .boolean label {
+    padding-left:20%;
+    padding-right: 10px;
+    width: 80%;
+  }
 
-    /* Hidden fields */
-    &.hidden {
+  fieldset > ol > li {
+    fieldset {
+      position:relative;
       padding: 0;
-    }
-
-    /* Errors */
-    p.inline-errors {
-      color: $error-color;
-      font-weight: bold;
-      margin:0.3em 0 0 20%;
-    }
-    ul.errors {
-      color: $error-color;
-      margin:0.5em 0 0 20%;
-      list-style:square;
-      li { padding:0; border:none; display:list-item; }
-    }
+      margin: 0;
 
-    &.error {
-      input[type=text], input[type=password], input[type=email], input[type=number], input[type=url], input[type=tel], textarea {
-        border: $border-width solid $error-color;
+      &.has_many_fields ol {
+        float: left;
+        width: 100%;
+        margin: 0;
       }
     }
   }
 
-  /* semantic_errors */
-  ul.errors {
-    background: lighten($error-color, 60%);
-    @include rounded(4px);
+  .inline-hints {
+    color: #999;
+    margin: 0.3em 0 0 20%;
+  }
+
+  .inline-errors, .errors {
     color: $error-color;
     font-weight: bold;
-    margin-bottom: 10px;
-    padding: 10px;
-    list-style:square;
-    li { margin-left:15px; padding:0; border:none; display:list-item; }
+    margin: 0.3em 0 0 20%;
   }
 
-  /* Buttons */
+  .errors {
+    background-color: lighten($error-color, 60%);
+    border-radius: 3px;
+    color: $error-color;
+    list-style-type: square;
+    margin: 0.5em 0 0 20%;
+    padding: 10px 10px 10px 25px;
+    width: calc(80% - 22px);
+  }
+
+  // Buttons
 
   input[type=submit], input[type=button], button {
     @include dark-button;
-    cursor: pointer;
-
   }
 
-  .buttons, .actions {
-    margin-top: 15px;
-    input[type=submit], input[type=button], button { margin-right: 10px; }
+  .button,
+  .action {
+    display: inline-block;
+    margin-right: 0.5em;
   }
 
-  fieldset.buttons li, fieldset.actions li {
-    float:left;
-    padding: 0;
-
-    &.cancel {
-      a {
-        @include light-button; @include icon(#777, 9px);
-      }
+  .cancel {
+    a {
+      @include light-button;
+      @include icon(#777, 9px);
```

Comment:
Color literals like `#777` should only be used in variable declarations; they should be referred to via variable everywhere else.